documentation/userguide/docs/shares.md

```diff
@@ -2,21 +2,23 @@
 Teams can browse data.all catalog and request access for data assets.
 data.all shares data between teams securely within and environment and across environments without any data movement.
 
+**Concepts**
 
-Datasets can contain tables and folders. Tables are Glue Tables registered in Glue Catalog.
-data.all uses (and automates)
-<a href="https://docs.aws.amazon.com/lake-formation/latest/dg/sharing-catalog-resources.html" target="_blank">Lake Formation sharing feature</a>
-to create access permissions to tables, meaning that no data is copied between  AWS accounts.
-
-Under-the-hood, folders are prefixes inside the dataset S3 bucket. To create sharing of folders in data.all,
-we create an S3 access point per requester group to handle its access to specific prefixes in the dataset.
+- Share request or Share Object: one for each dataset and requester team.
+- Share Item refers to the individual Redshift tables, Glue tables, folders or S3 Bucket that are added to the Share request.
```

Comment:
nit: make singular `Share Item refers to the individual Redshift table, Glue table, folder or S3 Bucket that is added to the Share request.`